.github/workflows/ci.yml

```
@@ -0,0 +1,40 @@
+name: CI
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
```

```suggestion
on:
  push:
    branches: [ "main" ]
  pull_request:
    branches: [ "main" ]
  workflow_dispatch:
```
I think it would be better to add `workflow_dispatch:` so that we can run this CI manually whenever we want to test.